### ISSUE TITLE
The corresponding TAB selected in the Design Pane should be selected in the Properties

### ISSUE THREAD

**Issue by atinprominic** [prominic/Moonshine-IDE#347]
OS: MAC

When the user switches from one TAB to another in the Design pane, the corresponding TAB name in the Properties panel should be selected and vice versa.

There may be the case where the user in the design pane selects the tab to be removed and clicks on remove button in properties panel. In this case a different tab which was selected in the properties panel is removed. This makes the user very confusing and results in the loss of his work due to undesired behaviour by Moonshine IDE.

**Comment by rat-moonshine** [comment]
Can you please elaborate more clearly (preferably with some image and steps) (?)

**Comment by atinprominic** [comment]
I have attached the video recording.

Note that in the recording, I am selecting the tabs named - "Title3" in the design pane, but the corresponding TAB in the Items section of Properties side bar is not selected. Also the Tab Title3 is not removed when I clicked on the Remove button in Properties side bar.
![pf090818](https://user-images.githubusercontent.com/34882311/43879350-50292308-9bc1-11e8-833b-8e6d2f9750e1.gif)
